.editorconfig

```diff
@@ -0,0 +1,248 @@
+root = true
+###############################
+# Core EditorConfig Options   #
+###############################
+# All files
+# [*]                            # Do not apply to all files not to break something
+# Either crlf | lf, default is system-dependent (when not specified at all)
+# end_of_line=crlf
+# Remove whitespace at the end of any line
+
+# Code files
+[*.{cs,csx,fs,fsx,vb,vbx}]
+trim_trailing_whitespace = true
+insert_final_newline     = true
+indent_style             = space # default=space
+indent_size              = 4     # default=4
+charset                  = utf-8
+
+# Project files
+[*.{csproj,fsproj,vbproj,shproj,projitems,props,plist}]
+trim_trailing_whitespace = true
+insert_final_newline     = true
+indent_style             = tab   # Visual Studio changes to tabs for old projects
+charset		             = utf-8
```

Comment:
Alignment?
```suggestion
charset		           = utf-8
```

Reply:
Oh wait, you're mixing tabs and spaces! Lol. We need an `.editorconfig` for the `.editorconfig` to prevent that from happening 😆.

Reply:
![image](https://user-images.githubusercontent.com/2365592/184755139-ad9f9982-823e-46e3-bec1-d0703cbff719.png)
Ah, here it is!